index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">  
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
```

Comment:
- [ ] kindly try to fix the extra white space and the margins like the previous user requested 

![Screenshot 2022-08-19 231810](https://user-images.githubusercontent.com/89282221/185714350-b66acb8f-d2b3-4207-a3de-ec71e74a2f5f.png)